G/GLEW/build_tarballs.jl

```diff
@@ -42,4 +46,4 @@ dependencies = [
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6")
```

Comment:
You need to change the version number to do this: #2763. If you can't change the version number, don't build for aarch64-apple-darwin

Reply:
Thanks for the heads up! I've bumped to v2.2.0